### DISCUSSION TITLE
Language switcher

### DISCUSSION THREAD

**Discussion by james-l-smith** [CollectionBuilder/collectionbuilder.github.io#63] [Q&A]
Hi. I was wondering how easy it would be to toggle languages. I saw the instructions for changing language in config.yml, but is there a way to move between e.g. config files in two languages?

**Comment by evanwill** [comment]
@james-l-smith toggling between multiple languages  it is not *super* easy... but is possible. How hard it is depends on how much of the UI you want to swap out for each language (e.g. the main nav, the titles, the metadata display names, the metadata fields, etc). There are a number of different approaches used in static sites, so how you implement it would depend on your goals and requirements. 

That said, we are interested in getting it to work and @owikle has been developing an approach in [cb-multilingual](https://github.com/owikle/cb-multilingual) to test ideas using JS to swap everything out. It is still drafty, but might give you an idea of what it could look like. 

If your collection is more static, another option is essentially setting up parallel versions for each language, basically like [translating CB into another language](https://collectionbuilder.github.io/cb-docs/docs/advanced/language/), so you would have a full config for each, and building them out in parallel. The different languages would be switched using the url (e.g. /en/subjects.html vs /es/subject.html).

**Reply by james-l-smith** [reply]
Thanks @evanwill! I think, given our needs, your idea of building it out twice and linking the two versions to each other on the top menu makes sense. There are some other text blocks I could put in each language in the spreadsheet, so it would have other benefits.

**Reply by evanwill** [reply]
Great, let us know how it goes!
